src/pages/MangaLib/tests.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "mangalib",
+  "url": "https://mangalib.org/",
+  "testCases": [
+    {
+      "url": "https://mangalib.org/ru/3757--ijiranaide-nagatoro-san/read/v18/c132",
```

Comment:
I think it was hit by a DCMA takedown

Reply:
Well.. you are right. Will change it. Local company got license to translate/sell manga so they removed it